cobigen-templates/templates-devon4j/src/main/templates/crud_java_server_app/templates/java/${variables.rootPackage}/${variables.component}/logic/api/to/${variables.entityName}Eto.java.ftl

```diff
@@ -2,15 +2,21 @@
 package ${variables.rootPackage}.${variables.component}.logic.api.to;
 
 import com.devonfw.module.basic.common.api.to.AbstractEto;
+import ${variables.rootPackage}.general.common.api.to.AbstractComposedKeyEto;
 import ${variables.rootPackage}.${variables.component}.common.api.${variables.entityName};
 
 import java.util.List;
 import java.util.Set;
 
+<#assign compositeIdTypeVar = JavaUtil.getReturnTypeOfMethodAnnotatedWith(classObject,"javax.persistence.EmbeddedId")>
+<#if compositeIdTypeVar!="null">
+import ${variables.rootPackage}.${variables.component}.common.api.${compositeIdTypeVar};
+</#if>
 /**
  * Entity transport object of ${variables.entityName}
  */
-public class ${variables.entityName}Eto extends <#if pojo.extendedType.canonicalName=="java.lang.Object" || pojo.extendedType.package!=pojo.package || pojo.extendedType.name == "ApplicationPersistenceEntity">AbstractEto<#else>${pojo.extendedType.name?replace("Entity","Eto")}</#if> implements ${variables.entityName} {
+public class ${variables.entityName}Eto extends <#if compositeIdTypeVar!="null">AbstractComposedKeyEto<${compositeIdTypeVar}><#elseif pojo.extendedType.canonicalName=="java.lang.Object" || pojo.extendedType.package!=pojo.package || pojo.extendedType.name == "ApplicationPersistenceEntity">AbstractEto<#else>${pojo.extendedType.name?replace("Entity","Eto")}</#if> implements ${variables.entityName} { 
```

Comment:
In case of a composed key, please use AbstractGenericEto from https://github.com/devonfw/devon4j/pull/242/files#diff-5bf762d93188bf156000c9543404ef7dR22

Reply:
Changes done and committed. Re-Review required.